src/containers/Chat/ChatMessages/ChatMessage/ChatMessage.tsx

```diff
@@ -22,9 +23,11 @@ export const ChatMessage: React.SFC<ChatMessageProps> = (props) => {
 
   return (
     <div className={[styles.ChatMessage, additionalClass].join(' ')}>
-      <div className={styles.Content} data-testid="content">
-        {props.body}
-      </div>
+      <Tooltip title={moment(props.insertedAt).format('MMM, DD YYYY')} placement="right">
```

Comment:
is this working or should be this instead?
```suggestion
      <Tooltip title={moment(props.insertedAt).format('MM, DD YYYY')} placement="right">
```

Reply:
Also, need @kurund's input on this. Which date format are we using? I think it should be consistent system-wide.

Reply:
The 'MMM' shows the first 3 letters of the month so it will look like Jan, Feb, Mar, etc. whereas 'MM' will show 1, 2, 3... 12. I can change this if we should be using a different format, please let me know what format you want! :)